src/config/ConfigManager.cpp

```diff
@@ -1647,15 +1651,20 @@ std::optional<std::string> CConfigManager::handleMonitor(const std::string& comm
     if (ARGS[2].starts_with("auto")) {
         newrule.offset = Vector2D(-INT32_MAX, -INT32_MAX);
         // If this is the first monitor rule needs to be on the right.
-        if (ARGS[2] == "auto-right" || ARGS[2] == "auto" || m_dMonitorRules.empty())
+        if (ARGS[2] == "auto-right" || ARGS[2] == "auto" || m_dMonitorRules.empty()) {
```

Comment:
???

Reply:
wait what's the issue

Reply:
no {} around short ifs, why did you add them

Reply:
I added a multiline else clause so I thought I should add braces around the previous if statements as well, is that not a thing?

Reply:
nope